### ISSUE TITLE
Can't instantiate class ColorOutputInfo

### ISSUE THREAD

**Issue by bolshakovsashka** [Clarifai/clarifai-java#36]
Hi, recently, after every request, i get an error:

```
Fatal Exception: java.lang.RuntimeException: Failed to invoke clarifai2.dto.model.output_info.ColorOutputInfo() with no args
       at com.google.gson.internal.ConstructorConstructor$3.construct(ConstructorConstructor.java:111)
       at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:210)
       at com.google.gson.Gson.fromJson(Gson.java:887)
       at com.google.gson.Gson.fromJson(Gson.java:952)
       at com.google.gson.Gson.fromJson(Gson.java:925)
       at clarifai2.internal.InternalUtil.fromJson(InternalUtil.java:145)
       at clarifai2.dto.model.output_info.OutputInfo$Adapter$1.deserialize(OutputInfo.java:37)
       at clarifai2.dto.model.output_info.OutputInfo$Adapter$1.deserialize(OutputInfo.java:23)
       at clarifai2.internal.JSONAdapterFactory$Adapter.read(JSONAdapterFactory.java:173)
       at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
       at com.google.gson.Gson.fromJson(Gson.java:887)
       at com.google.gson.Gson.fromJson(Gson.java:952)
       at com.google.gson.Gson.fromJson(Gson.java:925)
       at clarifai2.internal.InternalUtil.fromJson(InternalUtil.java:145)
       at clarifai2.dto.model.Model$Adapter$2.deserialize(Model.java:212)
       at clarifai2.dto.model.Model$Adapter$2.deserialize(Model.java:197)
       at clarifai2.internal.JSONAdapterFactory$Adapter.read(JSONAdapterFactory.java:173)
       at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
       at com.google.gson.Gson.fromJson(Gson.java:887)
       at com.google.gson.Gson.fromJson(Gson.java:952)
       at clarifai2.api.request.model.GetModelRequest$1$1.fromJSON(GetModelRequest.java:34)
       at clarifai2.api.request.model.GetModelRequest$1$1.fromJSON(GetModelRequest.java:32)
       at clarifai2.api.request.ClarifaiRequest$Impl.executeSync(ClarifaiRequest.java:285)
       at clarifai2.api.request.ClarifaiRequest$Builder.executeSync(ClarifaiRequest.java:192)
       at clarifai2.dto.model.DefaultModels.update(DefaultModels.java:52)
       at clarifai2.dto.model.DefaultModels.access$000(DefaultModels.java:10)
       at clarifai2.dto.model.DefaultModels$1.run(DefaultModels.java:42)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
Caused by java.lang.InstantiationException: can't instantiate class clarifai2.dto.model.output_info.ColorOutputInfo
       at java.lang.reflect.Constructor.constructNative(Constructor.java)
       at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
       at com.google.gson.internal.ConstructorConstructor$3.construct(ConstructorConstructor.java:108)
       at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:210)
       at com.google.gson.Gson.fromJson(Gson.java:887)
       at com.google.gson.Gson.fromJson(Gson.java:952)
       at com.google.gson.Gson.fromJson(Gson.java:925)
       at clarifai2.internal.InternalUtil.fromJson(InternalUtil.java:145)
       at clarifai2.dto.model.output_info.OutputInfo$Adapter$1.deserialize(OutputInfo.java:37)
       at clarifai2.dto.model.output_info.OutputInfo$Adapter$1.deserialize(OutputInfo.java:23)
       at clarifai2.internal.JSONAdapterFactory$Adapter.read(JSONAdapterFactory.java:173)
       at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
       at com.google.gson.Gson.fromJson(Gson.java:887)
       at com.google.gson.Gson.fromJson(Gson.java:952)
       at com.google.gson.Gson.fromJson(Gson.java:925)
       at clarifai2.internal.InternalUtil.fromJson(InternalUtil.java:145)
       at clarifai2.dto.model.Model$Adapter$2.deserialize(Model.java:212)
       at clarifai2.dto.model.Model$Adapter$2.deserialize(Model.java:197)
       at clarifai2.internal.JSONAdapterFactory$Adapter.read(JSONAdapterFactory.java:173)
       at com.google.gson.TypeAdapter$1.read(TypeAdapter.java:199)
       at com.google.gson.Gson.fromJson(Gson.java:887)
       at com.google.gson.Gson.fromJson(Gson.java:952)
       at clarifai2.api.request.model.GetModelRequest$1$1.fromJSON(GetModelRequest.java:34)
       at clarifai2.api.request.model.GetModelRequest$1$1.fromJSON(GetModelRequest.java:32)
       at clarifai2.api.request.ClarifaiRequest$Impl.executeSync(ClarifaiRequest.java:285)
       at clarifai2.api.request.ClarifaiRequest$Builder.executeSync(ClarifaiRequest.java:192)
       at clarifai2.dto.model.DefaultModels.update(DefaultModels.java:52)
       at clarifai2.dto.model.DefaultModels.access$000(DefaultModels.java:10)
       at clarifai2.dto.model.DefaultModels$1.run(DefaultModels.java:42)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:841)
```

After read stackowerflow, i update gson to 2.8.0, but i had same problem.

Version of Clarifai: 2.3.0
Version of Gson: 2.8.0

**Comment by sinnott-balin** [comment]
Same issue here.

**Comment by jared-clarifai** [comment]
Hey guys - looking into this one. In the mean time version 2.2.0 should be working.

**Comment by KevCron** [comment]
This should be resolved in 2.2.6

**Comment by KevCron** [comment]
I'm going to close this for now, if anyone still has this issue feel free to open a new one.
